### ISSUE TITLE
[changelog] Formatting issues with conventional commits preset 

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1189]
### Task List

- [x] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

A commit message such as

```
feat(core): Support Zstd compression in archive assemblers. Resolves  #1106
```

Generates an entry such as

```
84d9ae2 core: Support Zstd compression in archive assemblers. Resolves #1106, closes #106
```

Whereas a commit message such as

```
feat: Add init & json-schema commands to Ant, Maven, and Gradle.

Resolves #1123
```

Generates the following entry

```
d46c04e Add init & json-schema commands to Ant, Maven, and Gradle., closes #1123
```

1. `Resolves #1106, closes #106` should either be `Resolves #1106` or `Closes #106`
2. `Gradle., closes #1123` should be `Gradle. Closes #1123`

Other keywords that get picked up includes `Fixes`, `Relates`.

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.4.0
- **JDK Version:** N/A

**Comment by gotson** [comment]
I would argue that the commits are not exactly following the conventional commit spec. Issues being closed should be mentioned in a footer instead of the description. Description being one line, the `.` at the end is superfluous.

**Comment by aalmiray** [comment]
True. I wonder if a relaxed convention could be [conditionally] applied as GitHub adds issue/pr numbers to titles automatically and likely some people (like myself) will fail to follow strict CC conventions form time to time.

**Comment by gotson** [comment]
I don't think having a semantic parser to handle that in jreleaser makes sense.

I would suggest to use either one form or the other, and adapt the template to fit.

**Comment by aalmiray** [comment]
I was not thinking in a complex parser. Rather, if the title ends with `(#number)`. Only a single issue would be supported as this is the format suggested by GitHub. No other checks would be performed on the title before having ConventionalCommit perform its job.
